README.md

```diff
@@ -17,27 +17,46 @@ Want to use ROS, but don't want to run Ubuntu? This project uses the power of [N
 
 ## Examples
 
-Turtlebot 3 simulation in Gazebo:
+Turtlebot 3 simulation in Gazebo on noetic distro:
 ```
 nix-shell \
   -I nix-ros-overlay=https://github.com/lopsided98/nix-ros-overlay/archive/master.tar.gz \
   --option extra-substituters 'https://ros.cachix.org' \
   --option trusted-public-keys 'cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= ros.cachix.org-1:dSyZxI8geDCJrwgvCOHDoAfOm5sV1wCPjBkKL+38Rvo=' \
-  '<nix-ros-overlay/examples/turtlebot3-gazebo.nix>'
+  '<nix-ros-overlay/examples/noetic-turtlebot3-gazebo.nix>'
```

Comment:
Please don't change the names of existing files (and attributes below). Users may rely on these names in their Nix expressions and this would break them.

Reply:
I was discussing that with @mdrodrigo and he brought a valid point that the example was not working as nobody noticed that it was failing to build. So I'm pretty sure that users are not relying on this otherwise we would have received issues about it failing to build and run. Considering that I think that would be better to rename the attribute so it is clear which version it is using and causes less confusion for users when testing it.

What do you think?